domain/src/main/java/org/oppia/domain/question/QuestionDataController.kt

```diff
@@ -0,0 +1,83 @@
+package org.oppia.domain.question
+
+import android.content.Context
```

Comment:
Nit: unused import (I noticed this in Android Studio after branching off of your PR). Please remove.

Reply:
Done